taskfile/templates/default.yaml

```diff
@@ -0,0 +1,15 @@
+# https://taskfile.dev
+#
+# yaml-language-server: $schema=https://taskfile.dev/schema.json
```

Comment:
I'm a bit unsure about this one. It's not actually needed for `--init` because `Taskfile.yaml` is recognized by default. Not having it makes it clean.

On the other hand, it's useful to inform users that they can do this to keep the YAML LSP working for non-standard names. This would be useful for non-standard names used by `task -t foo.yaml` or Taskfiles added to `includes`.

Reply:
My two cents
I agree with @pd93 
I think taskfile.dev is not necessary, I'd even say it adds more noise than value. I’m also in favor of keeping the YAML server instruction.
Two days ago, I gave a talk about Task, I created the file with `task --init` and I had to serch in our website to find the YAML server instruction

Reply:
OK, I removed the `taskfile.dev` comment and added the YAML language server back.